src/components/Inputs/BirthdateInput/BirthdateInput.test.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import { BirthdateInput } from './BirthdateInput';
-import renderer from 'react-test-renderer';
```

Comment:
🤦‍♂ this is going to be a very hard habit to break lol. I think we gotta get eslint over into DS lol

Reply:
It's helpful IMHO (if you're using Sublime) to set up the Prettier plugin so you can format on save or with a key combo. https://github.com/getethos/ethos/blob/trunk/devops/dev/Sublime%20Text%203/JsPrettier.sublime-settings